api/v1/paths/pluginController/jsonFile.js

```diff
@@ -0,0 +1,38 @@
+const fs = require("fs");
+const path = require("path");
+const { processResponse } = require("../utils");
+
+module.exports = function () {
+    let operations = {
+        GET,
+    };
+
+    function GET(req, res, next) {
+        try {
+            var filePath = req.query.filePath;
+            var realPath = path.resolve(`public/vocables/${filePath}`);
+            var data = "" + fs.readFileSync(realPath);
```

Comment:
ça serait mieux de séparer API et accès aux données / système de fichiers, non ?

si on ne le fait pas maintenant, au moins mettre un // TODO move this to models/